src/pages/departures/[[...id]].tsx

```diff
@@ -0,0 +1,30 @@
+import DefaultLayout from '@atb/layouts/default';
```

Comment:
shouldn't the file just be named `[...id]`?

Reply:
https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes#optional-catch-all-segments

Reply:
Ohh. TIL! Makes sense!